src/main/resources/db/migration/V10__add_shipments_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS organisations_schema.shipments
+(
+    id          UUID    DEFAULT   uuid_generate_v4() PRIMARY KEY,
```

Comment:
Intentionally kept the schema simple - just to demonstrate for exercise requirement. Same is applicable to Order table.